src/Swarrot/Processor/SignalHandler/SignalHandlerProcessor.php

```diff
@@ -41,6 +41,10 @@ public function __construct(ProcessorInterface $processor, LoggerInterface $logg
      */
     public function setDefaultOptions(OptionsResolverInterface $resolver)
     {
+        if (!extension_loaded('pcntl')) {
+            return;
```

Comment:
this is not. The ``signal_handler_signals`` option should still be know by the OptionsResolver to prevent exceptions when using it because of an unknown option